src/audio/pipeline.c

```diff
@@ -305,6 +304,17 @@ static int pipeline_comp_params(struct comp_dev *current, void *data, int dir)
 	/* send current params to the component */
 	current->params = ppl_data->params->params;
 
+	/* set frames from samplerate/period, but round integer up */
```

Comment:
We can use ALIGN_UP macro.

Reply:
ok

Reply:
hmm so I want the rate to be divisible with frames, but change the frames (not rate)... can we still use align up?

Reply:
@juimonen It can stay as it is.